app/components/post_list/post_list.js

```diff
@@ -154,12 +155,17 @@ export default class PostList extends PureComponent {
         } = this.props;
         const {managedConfig} = this.state;
 
+        const highlight = highlightPostId && highlightPostId === postId;
```

Comment:
Why do we need to check for existence and equality? Wouldn't equality just be enough?

Reply:
That was added by HH I just moved it to use it elsewhere